pycbc/workflow/injection.py

```diff
@@ -226,7 +227,13 @@ def setup_injection_workflow(workflow, output_dir=None,
             inj_job = exe(workflow.cp, inj_section_name,
                           out_dir=output_dir, ifos='HL',
                           tags=curr_tags)
-            node = inj_job.create_node(full_segment)
+            if exe is PycbcCreateInjectionsExecutable:
```

Comment:
Does this actually work, or did you intend to use `isinstance()`?

Reply:
I probably did intend to use `isinstance()` but this does work

Reply:
I am not convinced:
```
In [1]: class Test:
   ...:     def __init__(self):
   ...:         self.foo = 12
   ...: 

In [2]: x = Test()

In [3]: isinstance(x, Test)
Out[3]: True

In [4]: x is Test
Out[4]: False
```

Reply:
Ah sorry, `select_generic_executable()` returns the actual class, not an instance. So yeah, this would work.